Doc/library/ast.rst

```diff
@@ -140,6 +140,11 @@ Node classes
    In the meantime, instantiating them will return an instance of
    a different class.
 
+.. note::
+    The descriptions of the specific node classes displayed here
+    were initially adapted from the fantastic Green Tree Snakes `Green Tree
```

Comment:
```suggestion
    were initially adapted from the fantastic `Green Tree
```

Putting the name once is enough :wink: 

Other than this, it looks good, thank-you!

Reply:
Haha, ResT links will be the end of me 😅